httpcache_test.go

```diff
@@ -1,130 +0,0 @@
-package httpcache_test
```

Comment:
IMHO we should keep the tests and check they still pass (except the one specific to the config).